src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -649,6 +649,92 @@ public void TestNormalizeTuples()
             TestNormalizeDeclaration("public (string prefix,string uri)Foo()", "public (string prefix, string uri) Foo()");
         }
 
+        [Fact]
```

Comment:
Tests are copy from IDE tests, except 2 tests that produce an invalid tree and have different behavior (which I think is acceptable):

https://github.com/dotnet/roslyn/blob/6da1274c9d24c2f90a48290394a951b23617f2a3/src/Workspaces/CSharpTest/Formatting/FormattingTests_FunctionPointers.cs#L88-L122